client/azure_client_set.go

```diff
@@ -44,6 +45,20 @@ type AzureClientSet struct {
 	VnetPeeringClient *network.VirtualNetworkPeeringsClient
 }
 
+func init() {
```

Comment:
I'm not a huge fan of using `init()`. I would move it to `NewAzureClientSet`, but I'm not blocking. Just a comment.

Reply:
I'd like to keep it in `init()` because it's effectively modifying the global variable from `autorest` package. Therefore it should be only once, although the later iterations are no-ops.